### DISCUSSION TITLE
pi-Base community call Fri 2024 Dec 13 at 4pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#846] [Announcements]
Agenda

- Using the suggestion feature on PRs for short suggestions
- Assigning reviewers for PRs
- reduce buraucracy and improve overall efficiency (https://github.com/pi-base/data/issues/1055#issuecomment-2533584941)
- https://github.com/pi-base/web/pull/177#issuecomment-2453839804
- New welcome thread: https://github.com/orgs/pi-base/discussions/880 (linked from the "Contributing" button on website)
- Use of advanced theories/concepts unknown to pi-base in pi-base proofs (when alternatives are/are not available) (#1029)
- pi-Base @ JMM: https://meetings.ams.org/math/jmm2025/meetingapp.cgi/Paper/38809

Zoom link: https://southalabama.zoom.us/j/93833246214

Previous meeting: https://github.com/orgs/pi-base/discussions/788

Next meeting: https://github.com/orgs/pi-base/discussions/1076

**Comment by prabau** [comment]
Carolina conflict

**Reply by StevenClontz** [reply]
And I have a personal conflict as well anyway. Dec 13 works for me though.

**Reply by ccaruvana** [reply]
Sorry for commenting so late. I unfortunately have a personal conflict with the 13th, but not the following Friday.

**Reply by StevenClontz** [reply]
I won't be available on the 20th, so I think we will stick to the 13th. Looking forward to having you back in the loop soon though Chris - we've picked up some great contributors in the past few weeks.

**Reply by ccaruvana** [reply]
Sure thing. I've caught a glimpse of the activity, but haven't had the time to contribute meaningfully. I look forward to being a bit more active in the new year.

**Comment by prabau** [comment]
@StevenClontz Hi Steven.  Not sure if you missed this, but can we give @david20000813 write access to the official repo?  See comments at the end of #933.

**Reply by StevenClontz** [reply]
Sorry, I've been traveling! Will fix this now. Thanks @david20000813 for all the recent contributions; sorry I haven't had time to chime in myself.

**Reply by david20000813** [reply]
@StevenClontz Hi Steven! No worries. I’ve received and accepted the invitation now. And thanks to @prabau for the help in getting me familiar with the contribution process!

**Comment by prabau** [comment]
Removing from discussion the minor issue I had added:
- new issue templates at https://github.com/pi-base/data/issues/new/choose

And adding the more important issue discussed in #1029:
- Use of advanced theories/concepts unknown to pi-base in pi-base proofs (when alternatives are/are not available)
